pkg/http/server.go

```diff
@@ -208,6 +209,13 @@ func (s *server) useMetrics(next fasthttp.RequestHandler) fasthttp.RequestHandle
 	return next
 }
 
+func (s *server) apiLogging(next fasthttp.RequestHandler) fasthttp.RequestHandler {
+	return func(ctx *fasthttp.RequestCtx) {
+		log.Debugf("Dapr http API logging: , %s", string(ctx.Path()))
```

Comment:
For HTTP, in addition to path, also include the HTTP Method (GET, POST, ETC).

Please, use a simpler format: "HTTP API Called: %s %s"

/cc @yaron2  @msfussell